app/Models/Tracer/ProfilAlumni.php

```
@@ -6,11 +6,27 @@
 {
     use HasFactory, SoftDeletes;
 
+    protected static function booted(): void
+    {
+        static::saved(function ($alumni) {
+            Cache::forget('portal_total_alumni');
+            Cache::forget('portal_welcome_alumni_data');
+            Cache::forget('portal_welcome_alumni_stats');
+        });
+
+        static::deleted(function ($alumni) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$alumni'. (undefined)

(UnusedFormalParameter)